app/views/state_file/questions/name_dob/edit.html.erb

```diff
@@ -22,7 +28,13 @@
     </div>
     <% if f.object.ask_spouse_name?  %>
       <div class="blue-group">
-        <p><%= t(".spouse_name", last_four_ssn: current_intake.direct_file_data.spouse_ssn&.last(4)) %></p>
+        <p>
+          <% if f.object.ask_primary_dob? %>
```

Comment:
should this be `.ask_spouse_dob?` instead of `.ask_primary_dob`?

Reply:
@bengolder good catch! fixed it